content/english/sql_workshop/activity3/_activity3.css

```
@@ -0,0 +1,76 @@
+.terminal_div {
+	position:relative;
+	height:34vw;
+	width: auto;
```

Spaces here instead of tabs for consistency:

```suggestion
  position:relative;
  height:34vw;
  width: auto;
```